website/post-process/walk.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Read directory for files.
+ * @see https://stackoverflow.com/a/5827895/1526789
+ */
+let walk = function(dir, done) {
```

Comment:
should be a `const` as you never override it.

Reply:
You are correct. Both these was fixed in https://github.com/async-aws/aws/pull/461